spark-connector/src/main/java/com/datastrato/gravitino/spark/connector/hive/HivePropertiesConverter.java

```diff
@@ -98,11 +108,21 @@ static Map<String, String> toOptionProperties(Map<String, String> properties) {
                     return TableCatalog.OPTION_PREFIX
                         + key.substring(
                             HivePropertiesConstants.GRAVITINO_HIVE_SERDE_PARAMETER_PREFIX.length());
+                  } else if (key.equals(HivePropertiesConstants.GRAVITINO_HIVE_TABLE_TYPE)) {
```

Comment:
please don't mix the external logic and option process logic, it's hard to maintain.

Reply:
@charliecheng630 could you fix this?  move `external` logic from `toOptionProperties`